config_complete.toml

```diff
@@ -153,6 +153,33 @@ AdditionalErrorFields = [
 # Eligible for live reload.
 # CacheOverrunStrategy = "impact"
 
+############################
+## Event ID Field Names   ##
+############################
+
+# Set the field names to use for the event ID fields. These fields are used to
+# identify events that are part of the same trace. The default values are
+# "trace.trace_id", "trace.span_id", "trace.parent_id". These values are 
```

Comment:
You set the defaults properly above, but this explanation is incomplete; please add traceId and parentId to the defaults.